src/StateNodeViz.scss

```
@@ -64,6 +64,7 @@
 
 [data-viz='stateNode-content'] {
   background: var(--viz-node-color-bg);
+  padding: 0.6rem 1.4rem;
```

```suggestion
  padding: 0.5rem 1.5rem;
```

A little less magical 😅  (So that our "unit" can be 0.5rem)